front/src/apollo/apolloProvider.ts

```diff
@@ -17,6 +43,18 @@ export const createClient = new ApolloClient({
               return [...incoming];
             },
           },
+          chat: {
+            merge(existing, incoming) {
+              const res = {};
+              if (existing !== undefined) {
+                Object.assign(res, existing);
+              }
+              if (incoming !== undefined) {
```

Comment:
```suggestion
              } else if (incoming !== undefined) {
```

따로 if로 분리하신 이유가 있으신지 궁금합니다.

Reply:
existing, incoming이 둘 다 존재할 경우를 위해 else if 대신 if를 사용하였습니다.